### ISSUE TITLE
roslaunch gseg_benchmark gseg_benchmark.launch alg:=all seq:=01

### ISSUE THREAD

**Issue by L00qqq** [url-kaist/Ground-Segmentation-Benchmark#12]
When I “ roslaunch gseg_benchmark gseg_benchmark.launch alg:=all seq:=01 ”, the following issues occurred:“       ERROR: cannot launch node of type [gseg_benchmark/common.sh]: Cannot locate node of type [common.sh] in package [gseg_benchmark]. Make sure file expermission is set to executable (chmod +x)          ”; i can not find the file common.sh,  May I ask what the reason is,,,,,,,,

**Comment by LimHyungTae** [comment]
Type `sudo chmod -R 777 {target shell file}`, then please rerun it.&nbsp;


Thx.


-----Original Message-----From:  "L00qqq" ***@***.***&gt;To:     "url-kaist/Ground-Segmentation-Benchmark" ***@***.***&gt;; Cc:     "Subscribed" ***@***.***&gt;; Sent:  2023-11-30 (Th) 17:03:30 (UTC+09:00)Subject: [url-kaist/Ground-Segmentation-Benchmark] roslaunch gseg_benchmark gseg_benchmark.launch alg:=all seq:=01 (Issue #12)

When I “ roslaunch gseg_benchmark gseg_benchmark.launch alg:=all seq:=01 ”, the following issues occurred:“       ERROR: cannot launch node of type [gseg_benchmark/common.sh]: Cannot locate node of type [common.sh] in package [gseg_benchmark]. Make sure file expermission is set to executable (chmod +x)          ”; i can not find the file common.sh,  May I ask what the reason is,,,,,,,,

**Comment by L00qqq** [comment]
Thank you very much for your prompt reply. It has been resolved. Great work, thank you very much

**Comment by fwailiu** [comment]
Hello, I'm typing roslaunch gseg_benchmark gseg_benchmark.launch alg：=all seq：=01，rivz no image appears, how can I fix it?

**Comment by LimHyungTae** [comment]
Please just go to `rviz` folder and run `rviz -d ${NAME}.rviz`

**Comment by fwailiu** [comment]
Thank you very much for your reply.

**Comment by fwailiu** [comment]
Hello, may I ask you, how did you do the experiments in the patchwork paper table1 about the three parts of R-GPF+U, CZM+U, CZM+U+E, is it also in the ground seg. benchmark code?

**Comment by LimHyungTae** [comment]
No, you have to manually test them. But I think the experimental numbers are not exactly correct, i.e. it is bit hard to reproduce the code since we modify the pipelines and evaluation a little bit (might not be a big problem)
